js/components/portal-dashboard/custom-select.tsx

```diff
@@ -53,8 +54,9 @@ export class CustomSelect extends React.PureComponent<IProps, IState> {
   }
 
   private renderHeader = () => {
-    const { items, HeaderIcon } = this.props;
-    const currentItem = items.find(i => i.action === this.state.current);
+    const { items, HeaderIcon, selectState } = this.props;
+    const selectedItem = selectState ? selectState : this.state.selected;
+    const currentItem = items.find(i => i.action === selectedItem);
```

Comment:
I find the naming here somewhat confusing. `selectState` is a prop not state. `selectedItem` appears to be an action not an item. The `currentItem` is presumably also the selected item.